src/components/Elements/Basic.tsx

```diff
@@ -41,34 +44,56 @@ const Basic: FunctionComponent<Props> = (props) => {
     titleStyle,
     tooltip,
     tooltipStyle = {},
+    tooltipFollowCursor,
     altId,
     continuing,
   } = props;
+
+  const defaultTooltipStyle: CSSProperties = {
```

Comment:
Is this the original CSS from the `rt-element__toltip` class?

Reply:
yep, I copied the attributes that aren't positional, since the new tooltip handles positioning